scripts/run_local_environment.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+SCRIPTS_DIR=$(dirname $0)
+TARGET_DIR="${SCRIPTS_DIR%/*}"/scalecube-configuration-benchmarks/target
+JAR_FILE=$(ls ${TARGET_DIR} | grep -m 1 .jar)
+
```

Comment:
maybe let's add a check on jar existence and if it will false run `mvn clean install -DskipTests`

Reply:
Added.